### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jhonnycano**
- resposne mispell
- todos comment

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -354,7 +353,7 @@ HTML template for the primer that follows:
 	    <input id="todo_complete" type="checkbox" <%= completed %>>
 	    <%= title %>
     </div>
-	</script>​
+	</script>?
```
Comment (jdreesen) [inline index.md]: I think this is a mistake

File: index.md [hunk 2]
```diff
@@ -8460,7 +8459,7 @@ You can use some variables in your ```View``` to represent the actual state of t
 
 **Diacritic.js**
 
-A plugin for Backbone.Paginator that replaces diacritic characters (ă,ş,ţ etc) with characters that match them most closely. This is particularly useful for filtering.
+A plugin for Backbone.Paginator that replaces diacritic characters (a,s,t etc) with characters that match them most closely. This is particularly useful for filtering.
```
Comment (jdreesen) [inline index.md]: These characters shouldn't be changed

**Comment by addyosmani** [comment]
@jhonnycano could you address @jdreesen's comments please?

File: index.md (hunk 2 above)
Comment (jhonnycano) [inline index.md]: my mistake, sorry

**Comment by addyosmani** [comment]
Thank you! :)

**Pull request merged**